packages/api/src/schemas/envConfig.ts

```diff
@@ -0,0 +1,51 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * The schema of TeamsFx configuration.
+ */
+export interface EnvConfig {
+  $schema?: string;
+  /**
+   * The Azure resource related configuration.
+   */
+  azure: {
+    /**
+     * The default subscription to provision Azure resources.
+     */
+    subscriptionId?: string;
+    /**
+     * The default resource group of Azure resources.
+     */
+    resourceGroupName?: string;
+    /**
+     * The location of created resource group if needed.
+     */
+    location?: string;
+    /**
+     * The tenant to create AAD app.
+     */
+    tenantId?: string;
+  };
+  /**
+   * The Teams App mainfest related configuration.
+   */
+  manifest: {
+    description?: string;
+    /**
+     * Configs to customize the Teams app manifest.
+     */
+    values: {
+      [k: string]: unknown;
+    };
+    [k: string]: unknown;
```

Comment:
The `menifest` config interface is not finalized yet, should we include this in the schema now?

Reply:
This is not for the finalized version but the M2's goal, it's open to be modified.